presto-main/src/main/java/com/facebook/presto/cost/DisjointRangeDomainHistogram.java

```diff
@@ -68,6 +69,9 @@
 public class DisjointRangeDomainHistogram
         implements ConnectorHistogram
 {
+    private static final long INSTANCE_SIZE = ClassLayout.parseClass(DisjointRangeDomainHistogram.class).instanceSize();
+    private static final long RANGE_SIZE = ClassLayout.parseClass(Range.class).instanceSize();
```

Comment:
Technically not entirely accurate, but there isn't a great way to measure it. The inner fields of `Range` are of type `Cut<C>` which is not publicly accessible, so we cannot call `ClassLayout.parseClass` on it.

Anyways, this will actually be updated in the histogram PR (#22365) after this merges because one of the commits in that PR replaces the `Range` class here instead with Presto's `Range` instead. I would opt to leave this as is for now anyways because of that PR and also the fact that this class usually has very little contribution to memory except for cases when users have large amounts of filters in a query (e.g. 10k values in an `IN` clause)